src/v/wasm/tests/wasm_fixture.h

```diff
@@ -74,3 +75,6 @@ class WasmTestFixture : public ::testing::Test {
     model::transform_metadata _meta;
     std::vector<ss::sstring> _log_lines;
 };
+
+// For using fake_schema_registry outside of WASM fixtures.
+std::unique_ptr<wasm::schema_registry> make_fake_schema_registry();
```

Comment:
hmm, maybe the fake schema registry can live with schema registry? cc @rockwotj

Reply:
I had suggested to @jcipar to move it to `src/v/schema_registry`. I'm not sure how we want to structure the HTTP service vs the internal functionality. Maybe we have `src/v/schema_registry/http`. IDK we should involve the enterprise team in this discussion too.

Reply:
cc @BenPope hi!